src/c_kzg_4844.c

```diff
@@ -53,8 +53,9 @@
 
 /** Internal representation of a polynomial. */
 typedef struct {
-    fr_t evals[FIELD_ELEMENTS_PER_BLOB];
-} Polynomial;
+    fr_t *evals;
+    uint64_t length;
```

Comment:
`size_t`?

Polynomial is internal so we don't need to adhere to a fixed size.

Reply:
I also prefer `size_t`. Fixed.